### PR TITLE
Modified the Image 

### PR DESCRIPTION
**Pull request by impiia**

### CODE REVIEW

File: mw-webapp/src/component/image/Image.module.scss [hunk 1]
```diff
@@ -3,4 +3,26 @@
...
+  transition: transform 0.3s ease;
+
+  &.zoomable {
+    cursor: pointer;
+  }
+}
+
+.zoomedOverlay {
+  position: fixed;
+  z-index: 1000;
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/image/Image.module.scss:15]: PLease look styles in other files. We using global var for such variable. Check everywhere

File: mw-webapp/src/component/image/Image.module.scss [hunk 2]
```diff
@@ -3,4 +3,26 @@
...
+  top: 0;
+  left: 0;
+  display: flex;
+  width: 100%;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+  background-color: rgba(0, 0, 0, 0.8);
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/image/Image.module.scss:23]: We don't use rgba in the project

File: mw-webapp/src/component/image/Image.tsx [hunk 3]
```diff
@@ -25,21 +27,65 @@ interface ImageProps {
...
+  };
+
+  const className = clsx(styles.image, props.className, props.zoomable && styles.zoomable);
+
+  /**
+   * Creates a portal for displaying a zoomed image overlay.
+   * If `isZoomed` is true, it renders an overlay with the zoomed image.
+   * Otherwise, it returns null.
+   */
+  const createZoomedPortal = () => {
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/image/Image.tsx:59]: We already have modal component? so better to use it

@tritonJS826  What do you think?

File: mw-webapp/src/component/image/Image.tsx [hunk 4]
```diff
@@ -25,21 +27,65 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enable image zooming on click
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/image/Image.tsx:32]: Add default value

File: mw-webapp/src/component/image/Image.tsx [hunk 5]
```diff
@@ -25,21 +27,65 @@ interface ImageProps {
...
+   */
+  zoomable?: boolean;
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/image/Image.tsx:34]: Boolean value start with is prefix (see other code)

File: mw-webapp/src/component/image/Image.tsx [hunk 6]
```diff
@@ -25,21 +27,65 @@ interface ImageProps {
...
 }
 
 /**
- * Сomponent for displaying images
+ * Component for displaying images
  */
 export const Image = (props: ImageProps) => {
-  const className = clsx(styles.image, props.className);
+  const [isZoomed, setIsZoomed] = useState(false);
```
Comment (Ekaterina1994) [inline mw-webapp/src/component/image/Image.tsx:41]: You don't need to use state in this case

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]

**Review by Ekaterina1994 — CHANGES_REQUESTED** [review]
Please read again our docs. Your PR's and commits  name does not comply with our conventions.

**Review by Ekaterina1994 — COMMENTED** [review]
Could you please tell us how did you pushed commit that don't comply with husky rules? Is it our bug?

**Review by Ekaterina1994 — COMMENTED** [review]
Please write your name on linkedin for possibility to connect

File: mw-webapp/src/component/image/Image.tsx (hunk 3 above)
Comment (tritonJS826) [reply mw-webapp/src/component/image/Image.tsx:59]: yeah, why not to use existent component? )

**Review by tritonJS826 — COMMENTED** [review]
Looks pretty good)

**Comment by tritonJS826** [comment]
> Could you please tell us how did you pushed commit that don't comply with husky rules? Is it our bug?

Yeah, use husky, please)

**Pull request closed without merging**